experimental/dds/sequence-deprecated/package.json

```diff
@@ -34,7 +47,7 @@
 		"prettier:fix": "prettier --write . --cache --ignore-path ../../../.prettierignore",
 		"test": "npm run test:mocha",
 		"test:coverage": "c8 npm test",
-		"test:mocha": "mocha --recursive \"dist/test/**/*.spec.js\" -r node_modules/@fluidframework/mocha-test-setup",
+		"test:mocha": "mocha --recursive \"dist/test/**/*.spec.*js\" -r node_modules/@fluidframework/mocha-test-setup",
```

Comment:
Wasn't sure if the extra '*' before 'js' was intentional normalization or a typo?

(In my local repo, I only see 3 other packages that use this pattern.)

Reply:
I think intentional, and I'm surprised you see so few, I thought we had already been moving to that in many places. Originally to accommodate for packages that hadn't moved to dual-emit and the ones that now produced `.cjs` files (easier to just have the pattern everywhere than to track them individually). Seems like overall we're going back to a `tsc` build for CJS that outputs `.js` files but I'm in favor of keeping this.

Reply:
Oh, and originally it was also not specifically for tests, but for code coverage settings and something else in package.json that I'm blanking on.